frontend/src/pages/ErrorsV2/ErrorsV2.tsx

```diff
@@ -234,6 +234,8 @@ export default function ErrorsV2() {
 			moreResults={getErrorsData.moreErrors}
 			resetMoreResults={getErrorsData.resetMoreErrors}
 			histogramBucketSize={getErrorsData.histogramBucketSize}
+			page={page}
```

Comment:
not sure if this is related to this PR, but the loading page is really wonky showing the page selectors loading when changing pages
https://www.loom.com/share/fe1bfe1f64014d10b6ff106382061b31

Reply:
https://www.loom.com/share/38fb9ec899b648779d353171e3115dce

Reply:
Taking a look! It seems like the new search passes in `totalCount=0` temporarily which causes the page controls to disappear